src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -75,9 +77,9 @@ public double getY(){
   private double kBaseI = 0.35;
   private double kBaseD = 0;
   //Elbow arm PID values
-  private double kElbowP = 1.5;
+  private double kElbowP = 1.3;
   private double kElbowI = 0.25;
-  private double kElbowD = 0;
+  private double kElbowD = 0.015;
```

Comment:
Didn't we revert these back at Waco to fix the arm control?

Reply:
No, we reverted a different change (Allowing negative outputs to come out of the elbow PID controller)